lib/facter/apache_version.rb

```diff
@@ -1,5 +1,4 @@
 Facter.add(:apache_version) do
-  confine kernel: ['FreeBSD', 'Linux']
```

Comment:
is it a good idea to remove this? Was this tested on other platforms? I got many modules that I've to monkeypatch because they have no confine statement, but were made for linux only and ruin puppet runs on my FreeBSD box. I assume the same could happen here on AIX / Solaris / whereever somebody wants to run this?

Reply:
Updated and added Darwin (line below) so the tests run on our machines locally :)